opennms-services/src/main/java/org/opennms/netmgt/notifd/BroadcastEventProcessor.java

```diff
@@ -447,7 +447,10 @@ private boolean continueWithNotice(Event event) {
         String service = event.getService();
 
         boolean continueNotice = false;
-
+        // If event is not associated with node, disregard Notice
+        if(nodeID == null) {
```

Comment:
What if the interface and/or service is null? Won't we run into the same problem?

Reply:
As @agalue indicated, service may not be critical. So I updated code to check node and interface. 
So the behavior now is:
If notice doesn't have valid node/interface, it won't be saved in DB.  
If only service is null, it  will still be saved in DB.